lib/board.rb

```diff
@@ -0,0 +1,5 @@
+class Board
+	def total_pieces
+      return 32
```

Comment:
No need to put `return` if it is the last line of the method

Reply:
Add Initialize method.